src/Avalonia.Controls/Platform/ExportAvaloniaModuleAttribute.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+
+namespace Avalonia.Platform
+{
+    [AttributeUsage(AttributeTargets.Assembly, AllowMultiple = true)]
+    public class ExportAvaloniaModuleAttribute : Attribute
+    {
+        public ExportAvaloniaModuleAttribute(string name, Type moduleType)
+        {
+            Name = name;
+            ModuleType = moduleType;
+        }
+
+        public string Name { get; private set; }
+        public Type ModuleType { get; private set; }
+
+        public string RequiredWindowingSubsystem { get; set; } = "";
```

Comment:
Ah I've just realized that this isn't for *changing* the windowing/rendering subsystem, it's for providing different modules for different subsystems, is that correct? That makes a lot more sense ;)